src/views/GameDetailView.vue

```diff
@@ -0,0 +1,73 @@
+<script setup>
+import { ref, onMounted } from 'vue';
+import { useGameLibraryStore } from '../stores/GameLibraryStore.js'; 
+import { useRoute } from 'vue-router';
+
+const route = useRoute();
+
+const gameLibraryStore = useGameLibraryStore();
+
+if(!gameLibraryStore.count) {
+  await gameLibraryStore.getGameDetails(route.params.id);
+}
+</script>
+
+<template>
+  <main class="game-detail-view">
+    <h1>Game Details</h1>
+
+    <h2>{{ gameLibraryStore.gameDetails.name }}</h2>
+
+    <img :src="gameLibraryStore.gameDetails.imageUrl" loading="lazy" />
+
+    <p class="game-description">{{ gameLibraryStore.gameDetails.description }}</p>
+
+    <ul class="game-info-list">
+      <li>
+        <b>min players: </b>{{ gameLibraryStore.gameDetails.minPlayers }}
+      </li>
+      <li>
+        <b>max players: </b>{{ gameLibraryStore.gameDetails.maxPlayers }}
+      </li>
+      <li>
+        <b>playing time: </b>{{ gameLibraryStore.gameDetails.playingTime }}
+      </li>
+    </ul>
+
+    <div class="category_container">
+      <span class="category" v-for="(category) in gameLibraryStore.gameDetails.categories" :key="index">
+        {{ category }}
+      </span>
+    </div>
+  </main>
+</template>
+
+<style>
+@import '@/assets/base.css';
+
+.game-detail-view {
+  margin: 10px;
+}
+
+.game-description {
+  margin-top: 10px;
+}
+
+.game-info-list {
+  margin: 10px;
+  list-style: none;
+}
+
+.category_container {
+  margin: 10px;
+  display: flex;
+}
+
+.category {
+  margin: 5px;
+  padding: 5px;
+  border-radius: 5px;
+  border: 4px solid var(--teal);
```

Comment:
to match the styling on the previous page, I'd recommend instead of setting the `border` property, 

I'd recommend change the background and color properties

```css
background: var(--teal);
color: white;
```